docker-compose.yml

```diff
@@ -28,32 +33,6 @@ services:
     ports:
       - "8100:80"
 
-  #               _
-  # __      _____| |__
-  # \ \ /\ / / _ \ '_ \
-  #  \ V  V /  __/ |_) |
-  #   \_/\_/ \___|_.__/
-  #
-
-  web:
-    image: metacpan/metacpan-web:latest
-    build:
-      context: ./src/metacpan-web
-    volumes:
-      - type: volume
-        source: web_carton
-        target: /carton
-      - type: bind
-        source: ./src/metacpan-web
-        target: /metacpan-web
```

Comment:
Where is this mapped now?

Reply:
It [mounts it at `/app`](https://github.com/metacpan/metacpan-web/blob/master/docker-compose.yml#L15), which is where the `Dockerfile` wants the code to be.

It also correctly has the `/app/local` volume to avoid interfering with the dependencies.